src/Okta.Sdk/Model/IdentityProvider.cs

```diff
@@ -125,7 +125,7 @@ public bool ShouldSerializeLastUpdated()
         /// Gets or Sets Links
         /// </summary>
         [DataMember(Name = "_links", EmitDefaultValue = true)]
-        public LinksSelf Links { get; set; }
+        public IdentityProviderLinks Links { get; set; }
```

Comment:
This is a breaking change right?  Will you update the version for this PR or are do you intend to submit another after merging this?

Reply:
Correct, I'm planning to address the release in a separate PR.